SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2356,3 +2357,8 @@ void USpatialNetDriver::SetSpatialDebugger(ASpatialDebugger* InSpatialDebugger)
 
 	SpatialDebugger = InSpatialDebugger;
 }
+
+FUnrealObjectRef USpatialNetDriver::GetCurrentPlayerControllerRef()
```

Comment:
I think it's worth checking that the `PackageMap` and the connection returned by `GetSpatialOSNetConnection` are valid.